### PR TITLE
read-only server

### PR DESCRIPTION
**Pull request by mtbc**
Effects https://trello.com/c/aCb5j0u4/24-document-omeroclusterreadonly by adding to https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ws/src/omero/_build/html/developers/Server/Clustering.html some notes on https://github.com/openmicroscopy/openmicroscopy/pull/5684. **Do not merge** until that PR is merged.

### CODE REVIEW

**Comment by hflynn** [comment]
Note that code is opened against metadata54 at the moment so this may not be ready for merging into develop even once that PR is merged.

**Comment by rleigh-codelibre** [comment]
Reads fine to me.

File: omero/developers/Server/Clustering.rst
Comment (manics) [inline omero/developers/Server/Clustering.rst]: `to do little more than` this sounds unduly negative, since it's actually really powerful functionality that let's you scale OMERO to as many servers as you want for data retrieval.

File: omero/developers/Server/Clustering.rst
Comment (manics) [inline omero/developers/Server/Clustering.rst]: In what situation would these be different?

**Review by manics — APPROVED** [review]
Reads OK, some suggested changes for you to consider.

File: omero/developers/Server/Clustering.rst
Comment (mtbc) [reply omero/developers/Server/Clustering.rst]: Ah, it was meant to sound reassuring, I clearly failed there!

File: omero/developers/Server/Clustering.rst
Comment (mtbc) [reply omero/developers/Server/Clustering.rst]: I don't much know the envisaged range of deployments but to give an idea of what I'd expect of the difference:

* with only DB being read-only, Bio-Formats probably doesn't need special configuration, it can keep on creating memo files in the main repo
* with only repo being read-only, users can probably still do things like tag images and draw ROIs on them.

**Comment by rleigh-codelibre** [comment]
The latest commits read fine to me and look good to merge.

**Comment by hflynn** [comment]
Holding off merging until the code PR is merged, unclear to me currently if this work is going into the 5.4.x line and we don't currently have another branch in the docs.

**Comment by joshmoore** [comment]
Merging for 5.4.6-rc1

**Pull request merged**
